app/scripts/lib/metrics.js

```diff
@@ -152,7 +152,7 @@ define([
       var url = this._collector + '/metrics';
       var self = this;
 
-      return this._send(filteredData, url)
+      return this._send(filteredData, url, ! flushSynchronously)
```

Comment:
@philbooth - While we are working in this module, would you mind if we did a tad bit of cleanup?

* Passing in the `url` to `_send` seems a bit silly, send is an instance function and should be able to create its own URL. Maybe the original intention was to send metrics to different endpoints for different situations. I don't remember.
* If a flush has already occurred and filteredData.events, filteredData.timers, and filteredData.ab are all empty, then we have no new information; pinging the metrics endpoint doesn't really make sense. I wonder if this could ever occur on the first send.